### PR TITLE
LRQA-77969 | master

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LRQA-77969

/cc @JoyceWang08211  @gaborlovasdotcom

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase [hunk 1]
```diff
@@ -107,6 +107,97 @@ definition {
...
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+
+			LAR.exportPortlet(
+				contentLastRange = "Last",
+				larFileName = "WebContent_portlet_structures.lar");
+		}
+
+		task ("When: The exported data is imported into a new virtual instance site") {
+			ApplicationsMenu.gotoPortlet(
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase:168]: Please use 
HeadlessPortalInstanceAPI.addPortalInstance(
				domain = "www.able.com",
				portalInstanceId = "www.able.com",
				virtualHost = "www.able.com");
to replace line168-176.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase [hunk 2]
```diff
@@ -107,6 +107,97 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-114230. Verify that can import image global structures between instances."
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase:110]: The test point is that assert the imported web content should show the image. This is different from the current description and case name. Please think over and modify the description and case name.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase [hunk 3]
```diff
@@ -107,6 +107,97 @@ definition {
...
+	@priority = "3"
+	test CanImportExportedImageGlobalStructuresBetweenInstances {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase:112]: Please modify the case name based on the previous requirement.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**